backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -94,7 +95,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
     // Find immediate match if possible
-    FindMatchQuery query = new FindMatchQuery(ZonedDateTime.now());
+    ZonedDateTime dateTime = ZonedDateTime.now();
+    Clock clock = Clock.fixed(dateTime.toInstant(), dateTime.getZone());
```

Comment:
Using a fixed clock still leaves the potential for inaccurate results from findMatch if the FindMatchQuery object exists for a long time. The key benefit of passing a clock is so that findMatch can be called at multiple times and always use the correct time when running the algorithm. It looks like the Clock.systemDefaultZone() or better Clock.systemUTC() will work here.

Reply:
Yep, I immediately realized my mistake right after pushing so I unfixed the clock in my "Unfix clock" commit above!

Reply:
Nice. Can't argue with a -4 minute turnaround on a PR comment